File: Beginner/Six_Odd_Numbers_1070/Main.java
package  Beginner.Six_Odd_Numbers_1070;

import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int number = sc.nextInt();
        int repeat_times = 6;

        if (number % 2 == 0) {
            number += 1;
        }
        for (int odd_number = number; odd_number < (number + (repeat_times * 2)); odd_number += 2) {
            System.out.println(odd_number);
        }
        sc.close();
    }
}

File: Beginner/Six_Odd_Numbers_1070/main.py
# number = int(input())
# odd_printed = 0

# while odd_printed < 6:
#     if number % 2 != 0:
#         print(number)
#         odd_printed += 1
#     number += 1

number = int(input())
repeat_times = 6
if number % 2 != 0:
    repeat_times = 5
    print(number)
    number += 2
else:
    number += 1

for odd_number in range(number, number+(repeat_times*2), 2):
    print(odd_number)
